docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+maintenance. Three or more virtual machines per availability set is recommended.
+
+Remember not to mix servers of different roles in the same availability set. For
+example, don’t place a ASCS node in the same availability set with the
+application server. When an availability set contains only virtual machines
+performing the same role, the Azure fabric tries to ensure that the services
+provided by the servers in the set remain available even during system failure.
+
+### Subnets
+Using multiple subnets, you can control different network security profiles in a
+consistent manner. For example, end users connect to SAP application servers
+during their normal business operations, but only administrators need access to
+the database servers. Place application servers on a separate subnet so you can
+secure them more easily by managing the subnet security policies, not the
+individual servers.
+
+### Load balancers
+[SAP Web Dispatcher][sap-dispatcher]. handles load balancing of HTTP(S) traffic to
+ dual-stack servers (ABAP and Java). SAP has advocated single-stack application servers for years, so very few
+applications run on a dual-stack deployment model nowadays. The Azure Load Balancer 
+depicted in the architecture diagram implements the high availability cluster for the SAP Web Dispatcher.
+
+For traffic from SAPGUI clients connecting a SAP server via DIAG and Remote
+Function Calls (RFC), the SCS message server balances the load by creating SAP
+App Server [Logon Groups][logon-groups], so no additional load balancer is needed. 
+
+### NICs
+SAP landscape management functions require segregation of server traffic on
+different NICs. For example, business data should be separated from
+administrative traffic and backup traffic. 
+Using multiple NICs to connect to different subnets provides the means for data
+segregation. For more information, see “Networking” in  [Building High Availability for SAP NetWeaver and SAP HANA][sap-ha].
+
+To implement traffic segregation in a virtual machine, each NIC should be
+connected to a separate subnet. For example, for the SAP application servers,
+the data communication NIC should be connected to the application server subnet,
+and the administration NIC should be connected to the management subnet. For
+configuration details, see [Multiple VM NICs and Network Virtual Appliances in Azure][multiple-vm-nics].
+
+### Network security groups
+We recommend connecting servers that perform the same role, such as application
+servers and database servers, to a single subnet. When a network security group
+is associated with a subnet, it then applies to all the servers within the
+subnet. To note exceptions, servers that require a special security profile may
+be connected to their own subnet and associated with a network security group
+within the group. For implementation details, see [Filter network traffic with network security groups][filter-network].
```

Comment:
This is a description of how NSGs work, but it's not a recommendation.  Do we have specific guidance here?